manywheel/build_common.sh

```diff
@@ -113,6 +99,7 @@ fi
 ########################################################
 # Compile wheels as well as libtorch
 #######################################################
+pushd "$PYTORCH_ROOT"
```

Comment:
Should we have a check to make sure that if this variable is empty that it errors out of the script before it does anything unexpected?